packages/create/templates/hello-drizzle/README.md

```diff
@@ -0,0 +1,48 @@
+# DBOS Hello with Drizzle
+
+This is a [DBOS app](https://docs.dbos.dev/) bootstrapped with `npx @dbos-inc/dbos-sdk init`.
```

Comment:
We already said "NOTE: This command has been removed in favor of `npx @dbos-inc/create` or `npm create @dbos-inc`"
So use the create command instead.

Reply:
We deprecated 'npm create' also, so the former one.
npx -y @dbos-inc/create@latest -n <app-name>